airflow/auth/managers/base_auth_manager.py

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from abc import abstractmethod
+
+from airflow.utils.log.logging_mixin import LoggingMixin
+
+
+class BaseAuthManager(LoggingMixin):
```

Comment:
Why does this need to inherit from LoggingMixin? It does not do any logging. I’d feel the classes that do the actual logging should inherit instead.

Reply:
Good point. I did it for the future since all sub classes will most likely need to log but I can remove it

Reply:
hmm yeah I think it's nice to inherit in base classes so that all the implems have it, but no hard opinion on it.